src/language-js/print/member-chain.js

```diff
@@ -106,7 +107,9 @@ function printMemberChain(path, options, print) {
     } else if (isMemberish(node)) {
       printedNodes.unshift({
         node,
+        path,
```

Comment:
`ASTPath` is a state machine, it make no sense to add to the array.

Reply:
Thank you, you're right.
I added it once and it didn't work. Then added `shouldInline` instead and forgot to remove it.